gadget/update.go

```diff
@@ -1620,8 +1620,8 @@ func resolveUpdate(oldVol *PartiallyLaidOutVolume, newVol *LaidOutVolume, policy
 			if err != nil {
 				return nil, err
 			}
-			// Nothing to do after filtering
-			if filter != nil && len(resolvedContent) == 0 && len(newStruct.LaidOutContent) == 0 {
```

Comment:
do you know why the check was like this? would there be a reason to go over things even if there's no content?